### PR TITLE
[Trivial] Fix double initialization of field errno in std/file.d

### PR DESCRIPTION
**Pull request by RazvanN7**
This is currently blocking: https://github.com/dlang/dmd/pull/8167

### CODE REVIEW

File: std/file.d [hunk 1]
```diff
@@ -172,15 +172,16 @@ class FileException : Exception
             msg  = Message describing the error.
             file = The _file where the error occurred.
             line = The _line where the error occurred.
+            errno = The error number; defaults to 0 if missing.
      +/
-    this(in char[] name, in char[] msg, string file = __FILE__, size_t line = __LINE__) @safe pure
+    this(in char[] name, in char[] msg, string file = __FILE__, size_t line = __LINE__, uint errno = 0) @safe pure
```
[the remark this answers is not in the document]
Comment (RazvanN7) [reply std/file.d]: how's this?

File: std/file.d [hunk 2]
```diff
@@ -172,15 +182,11 @@ class FileException : Exception
             msg  = Message describing the error.
             file = The _file where the error occurred.
             line = The _line where the error occurred.
+            errno = The error number; defaults to 0 if missing.
```
Comment (CyberShadow) [inline std/file.d]: No longer needed

Comment (RazvanN7) [reply std/file.d]: deleted! :dagger:

**Comment by RazvanN7** [comment]
@CyberShadow Thank you!

**Pull request merged**